### PR TITLE
colorcontrol: Add version 4.1.0.0

### PR DESCRIPTION
**Pull request by wangcheng**
[ColorControl](https://github.com/Maassoft/ColorControl) is a useful tool to change NVIDIA display settings and control LG TV.

### CODE REVIEW

File: bucket/colorcontrol.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": "4.1.0.0",
+    "description": "Easily change NVIDIA display settings and/or control LG TV's",
+    "homepage": "https://github.com/Maassoft/ColorControl",
```
Comment (issaclin32) [inline bucket/colorcontrol.json:4]:
```suggestion
    "homepage": "https://github.com/Maassoft/ColorControl",
    "license": "Freeware",
```

**Comment by issaclin32** [comment]
I have updated the manifest by myself because the PR seems outdated.

**Review by issaclin32 — APPROVED** [review]

**Comment by rashil2000** [comment]
image

It's a 64bit app. (very easy way to check - if the 'Compatibility mode' property starts from Vista, it's 64bit. If it starts from 95, it's 32bit.)

**Comment by issaclin32** [comment]
> It's a 64bit app. (very easy way to check - if the 'Compatibility mode' property starts from Vista, it's 64bit. If it starts from 95, it's 32bit.)

Fixed in https://github.com/ScoopInstaller/Extras/commit/bcf9360398b867e39c436d83e7fd3313e0bc9e72

**Pull request merged**